go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/viper v1.13.0
+	github.com/stretchr/testify v1.8.0
```

Comment:
I think this is overlapping a lot with gomega? I think we want to avoid extra 3p deps where possible. Is it easy to just use that instead like existing tests?

Reply:
I should have noticed that we were using a different assertion library. Done.